pkgs/tools/video/rwedid/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, rustPlatform
+, fetchFromGitea
+, pkg-config
+, xz
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rwedid";
+  version = "0.3.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "ral";
+    repo = "rwedid";
+    rev = version;
+    hash = "sha256-lQF6kAhb+qC7gDax5VL1NwZ7/sNyfGfJsoytoPEwkRg=";
+  };
+
+  cargoHash = "sha256-Xtdfl44kqn3v+NtSeWMsZnCAbxqHaIOxoNf2hGtDKXc=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    xz
+  ];
+
+  postInstall = ''
+    mkdir -p $out/etc/udev/rules.d
+    echo 'SUBSYSTEM=="i2c-dev",KERNEL=="i2c-[0-9]*", ATTRS{class}=="0x030000", TAG+="uaccess"' > $out/etc/udev/rules.d/60-ddcutil.rules
+  '';
+
+  meta = with lib; {
+    description = "Read and write EDID data over an I2C bus";
+    homepage = "https://codeberg.org/ral/rwedid";
+    license = licenses.mit;
```

```suggestion
    license = licenses.mit;
    platforms = platforms.linux;
```

Seems to be Linux-only software (as it requires a Linux kernel module).